DOC-697: New session variable setting for serial_normalization (#14460)
cockroachdb/docs: _includes/v22.1/misc/session-vars.md

| <a name="results-buffer-size"></a> `results_buffer_size` | The default size of the buffer that accumulates results for a statement or a batch of statements before they are sent to the client. <br/>This can also be set for all connections using the `sql.defaults.results_buffer_size` [cluster setting](cluster-settings.html). Note that auto-retries generally only happen while no results have been delivered to the client, so reducing this size can increase the number of retryable errors a client receives. On the other hand, increasing the buffer size can increase the delay until the client receives the first result row. Setting to 0 disables any buffering. | `16384` | Yes | Yes |
| <a name="require-explicit-primary-keys"></a> `require_explicit_primary_keys` | If `on`, CockroachDB throws on error for all tables created without an explicit primary key defined. | `off` | Yes | Yes |
| <a name="search-path"></a> `search_path` | A list of schemas that will be searched to resolve unqualified table or function names. <br/>For more details, see [SQL name resolution](sql-name-resolution.html). | `public` | Yes | Yes |

> | <a name="serial-normalization"></a> `serial_normalization` | Specifies the default handling of [`SERIAL`](serial.html) in table definitions. Valid options include `'rowid'`, `'virtual_sequence'`, `sql_sequence`, `sql_sequence_cached`, and `unordered_rowid`. <br/>If set to `'virtual_sequence'`, the `SERIAL` type auto-creates a sequence for [better compatibility with Hibernate sequences](https://forum.cockroachlabs.com/t/hibernate-sequence-generator-returns-negative-number-and-ignore-unique-rowid/1885). <br/>If set to `sql_sequence_cached`, the `sql.defaults.serial_sequences_cache_size` [cluster setting](cluster-settings.html) can be used to control the number of values to cache in a user's session, with a default of 256. <br/>If set to `unordered_rowid`, the `SERIAL` type generates a globally unique 64-bit integer (a combination of the insert timestamp and the ID of the node executing the statement) that does not have unique ordering. | `'rowid'` | Yes | Yes |

Reviewer: ```suggestion
| <a name="serial-normalization"></a> `serial_normalization` | Specifies the default handling of [`SERIAL`](serial.html) in table definitions. Valid options include `'rowid'`, `'virtual_sequence'`, `sql_sequence`, `sql_sequence_cached`, and `unordered_rowid`. <br/>If set to `'virtual_sequence'`, the `SERIAL` type auto-creates a sequence for [better compatibility with Hibernate sequences](https://forum.cockroachlabs.com/t/hibernate-sequence-generator-returns-negative-number-and-ignore-unique-rowid/1885). <br/>If set to `sql_sequence_cached`, you can use the `sql.defaults.serial_sequences_cache_size` [cluster setting](cluster-settings.html) to control the number of values to cache in a user's session, with a default of 256. <br/>If set to `unordered_rowid`, the `SERIAL` type generates a globally unique 64-bit integer (a combination of the insert timestamp and the ID of the node executing the statement) that does not have unique ordering. | `'rowid'` | Yes | Yes |
```
